sql/driver/wire.go

```diff
@@ -18,42 +18,92 @@
 package driver
 
 import (
+	"database/sql/driver"
 	"fmt"
-	"strconv"
 	"time"
+
+	"github.com/cockroachdb/cockroach/util"
 )
 
+var _ driver.Valuer = Datum{}
+
 const (
 	// Endpoint is the URL path prefix which accepts incoming
 	// HTTP requests for the SQL API.
 	Endpoint = "/sql/"
-
-	timestampWithOffsetZoneFormat = "2006-01-02 15:04:05.999999999-07:00"
```

Comment:
@vivekmenezes note I removed this - there was no test checking it, and AFAICT it's only used in `(driver.Datum).String()` which isn't really used anywhere. I added a test in wire_test.go, PTAL.